WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -20,7 +20,3 @@
     </div>
   </div>
 </div>
-
-<script>
-  $('#competitions-<%= params[:display] == "list" ? "map" : "list" %>').hide();
-</script>
```

Comment:
Really? That's what I'm trying to do over in WcaOnRails/app/views/competitions/index.js.erb. Did I miss something?